packages/interbit-ui-components/src/tests/uiKit.test.js

```diff
@@ -28,6 +29,82 @@ const countChildren = (parent, child, count) => {
   expect(wrapper.find(child).length).toBe(count)
 }
 
+const equalInnerText = (parent, child, text) => {
+  const wrapper = shallow(parent)
+  expect(wrapper.find(child).text()).toEqual(text)
+}
+
+const hasPropertyEqualTo = (parent, child, property, value) => {
+  const wrapper = shallow(parent)
+  expect(wrapper.find(child).prop(property)).toEqual(value)
+}
+
+describe('<ActivityBar />', () => {
+  let props = {}
+  beforeEach(() => {
+    props = {}
+    Object.assign(props, {
+      breadcrumb: [
+        {
+          title: 'First Level',
+          clickHandler: () => {}
+        }
+      ],
+      dateTimeFormat: 'YYYY MM DD',
+      firstName: 'John',
+      lastName: 'Doe',
+      timestamp: 1537852467,
+      userClickHandler: () => {}
+    })
+  })
+
+  it('renders a bredcrumb', () => {
+    countChildren(<ActivityBar {...props} />, 'div.breadcrumbs', 1)
+  })
+
+  it('It renders a timestamp in the specified format', () => {
+    equalInnerText(<ActivityBar {...props} />, 'div.date-time', '2018 09 25')
```

Comment:
This test fails in my timezone because you're converting the unix timestamp to the user's local timezone in the component. Please fix the test to make it clear that you are rendering the date and time in the user's timezone.